lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -83,87 +62,12 @@ export default function FilePickerApp({
     /** @type {ErrorInfo|null} */ (null)
   );
 
-  // Initialize the Google Picker client if credentials have been provided.
-  // We do this eagerly to make the picker load faster if the user does click
-  // on the "Select from Google Drive" button.
-  const googlePicker = useMemo(() => {
-    if (!googleClientId || !googleDeveloperKey || !googleOrigin) {
-      return null;
-    }
-    return new GooglePickerClient({
-      developerKey: googleDeveloperKey,
-      clientId: googleClientId,
-
-      // If the form is being displayed inside an iframe, then the backend
-      // must provide the URL of the top-level frame to us so we can pass it
-      // to the Google Picker API. Otherwise we can use the URL of the current
-      // tab.
-      origin: window === window.top ? window.location.href : googleOrigin,
-    });
-  }, [googleDeveloperKey, googleClientId, googleOrigin]);
-
   /**
    * Flag indicating whether the form should be auto-submitted on the next
    * render.
    */
   const [shouldSubmit, submit] = useState(false);
```

Comment:
You're right that this flag is currently not needed. However it will be needed again soon when introducing a new step to the assignment configuration flow to choose the groups configuration.